joomla/joomla-cms
# Fix so when creating a user with a username of more than 150 characters you get a useful error

Pull Request for Issue #28739 .

### Summary of Changes
Added info to the error message to explain the length limit for the username


### Testing Instructions
1) Go to https://<mydomain>/administrator/index.php?option=com_users&view=user&layout=edit
2) Enter details...
Name: test 
Login Name: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
Password: anything
Confirm Password: anything
Email: test@example.com
3) Click "Save"


### Expected result
The error message...
> Please enter a valid username. No space at beginning or end, at least %d characters, must <strong>not</strong> have the following characters: < > \ &quot; ' &#37; ; ( ) & and be less than 150 characters long.


### Documentation Changes Required
I don't think so, looking at https://docs.joomla.org/Help39:Users_User_Manager
